.nvmrc

```
@@ -0,0 +1 @@
+18.16.0
```

It's better to use an alias instead of a specific version. There is no need to pin a specific point release. I think v18 is `hydrogen`.

```suggestion
lts/hydrogen
```

---

I didnt know that, thanks.